docs/secure-connections/sauce-connect/proxy-tunnels.md

```diff
@@ -38,7 +38,7 @@ You can manage and monitor all Sauce Connect Proxy tunnel activity from the Sauc
 | :--- | :--- |
 | Type | The icon shows whether the tunnel is a Sauce Connect Proxy tunnel, or an IPSec VPN tunnel. |
 | State | The icon shows whether the tunnel is running or stopped. |
-| Tunnel Name | The name of the tunnel. This is the [`--tunnel-name`](/dev/cli/sauce-connect-proxy/#--tunnel-name) used when starting the Sauce Connect tunnel. |
+| Tunnel Name | The name of the tunnel. This is the [`--tunnel-identifier`](/dev/cli/sauce-connect-proxy/#--tunnel-name) used when starting the Sauce Connect tunnel. |
```

Comment:
See this comment: https://github.com/saucelabs/sauce-docs/pull/993/files#r784475982